synapse/handlers/room.py

```diff
@@ -1046,6 +1046,18 @@ async def send(etype: str, content: JsonDict, **kwargs: Any) -> int:
             if power_level_content_override:
                 power_level_content.update(power_level_content_override)
 
+            # override default_power_level_content_override for this room preset, if any
+            default_power_level_content_override = (
+                self.config.room.default_power_level_content_override
+            )
+            if (
+                default_power_level_content_override
+                and default_power_level_content_override.get(preset_config)
+            ):
+                power_level_content.update(
+                    default_power_level_content_override.get(preset_config)
+                )
+
```

Comment:
The documentation for `default_power_level_content_override` says that it's applied before `power_level_content_override`, but here it is the other way around?